core/src/main/java/org/apache/accumulo/core/client/impl/DelegationToken.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.accumulo.core.client.security.tokens;
+package org.apache.accumulo.core.client.impl;
```

Comment:
This invalidates a bit of documentation within the user manual on the subject. I thought you were going to still have a public API delegation token but move the sensitive stuff to an impl. It seems very confusing to me to have the notion of this special token implementation which users should/'may want to' be aware of,  but we say to the user "you don't get to know about it or reference it".